CryptKeeperZK/crypt-keeper-extension
# feat: add options page

## Explanation

This PR adds settings page and improves current popup.

Details are below:
- [x] Fix routing redirect to support options page
- [x] Add notification for removing all the history
- [x] Move history tracking and clear to general settings
- [x] Add redirects for login and onboarding pages

## More Information

Closes #45 

## Screenshots/Screencaps

### Before

![image](https://user-images.githubusercontent.com/14254374/233445817-ae83f2cf-85e0-4cd8-885c-088717129bfe.png)

### After

![image](https://user-images.githubusercontent.com/14254374/233445320-a7191719-5743-4cf6-bf2e-75bf130df08e.png)
![image](https://user-images.githubusercontent.com/14254374/233445381-597dbdae-13ce-4a32-b510-bd050396272a.png)
![image](https://user-images.githubusercontent.com/14254374/233446243-8b974bd3-b000-434c-b8db-27ab8a4d4424.png)

## Manual Testing Steps

1. Open menu from home page
2. Go to settings page
3. Clear history and toggle track history button
4. Check activity tab after removing or creating identities

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
